scripts/installOpenBLAS.ps1

```diff
@@ -2,16 +2,16 @@ Write-Host 'script installOpenBLAS.ps1 started'
 New-Item -Path 'C:\BLAS' -ItemType Directory -Force # create directory
 # Enforce stronger cryptography
 [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12
-$uri = 'https://github.com/xianyi/OpenBLAS/archive/v0.3.10.zip'
-$output = 'C:\BLAS\v0.3.10.zip'
+$uri = 'https://github.com/xianyi/OpenBLAS/archive/v0.3.12.zip'
+$output = 'C:\BLAS\v0.3.12.zip'
 $webclient = New-Object System.Net.WebClient
 $webclient.DownloadFile($uri,"$output")
-Expand-Archive -Path 'C:\BLAS\v0.3.10.zip' -DestinationPath 'C:\BLAS\OpenBLAS-v0.3.10' -Force # expand zip file
+Expand-Archive -Path 'C:\BLAS\v0.3.12.zip' -DestinationPath 'C:\BLAS\OpenBLAS-v0.3.12' -Force # expand zip file
 C:\Users\travis\build\AMICI\scripts\compileBLAS.cmd
 New-Item -Path 'C:\BLAS\lib' -ItemType Directory -Force # create directory
-Copy-Item "C:\BLAS\OpenBLAS-v0.3.10\OpenBLAS-0.3.10\lib\Release\openblas.lib" -Destination "C:\BLAS\lib" -Recurse
+Copy-Item "C:\BLAS\OpenBLAS-v0.3.12\OpenBLAS-0.3.12\lib\Release\openblas.lib" -Destination "C:\BLAS\lib" -Recurse
 New-Item -Path 'C:\BLAS\bin' -ItemType Directory -Force # create directory
-Copy-Item "C:\BLAS\OpenBLAS-v0.3.10\OpenBLAS-0.3.10\lib\openblas.dll" -Destination "C:\BLAS\bin" -Recurse
+Copy-Item "C:\BLAS\OpenBLAS-v0.3.12\OpenBLAS-0.3.12\lib\openblas.dll" -Destination "C:\BLAS\bin" -Recurse
```

Comment:
Might be good to introduce an `$openblas_version` variable in all the scripts. Will simplify future changes.